### DISCUSSION TITLE
GraphiQL instead of Playground?

### DISCUSSION THREAD

**Discussion by frederikhors** [async-graphql/async-graphql#1008] [Q&A]
A little question because I cannot find anything about this on this repo.

Are we going to replace `Playground` with `GraphiQL` sooner or later?

https://github.com/graphql/graphql-playground/issues/1366#issuecomment-1062088978

**Comment by maraisr** [comment]
+1 GraphiQL also has first-class support for the defer/stream RFC's already, so bit of a feature unlock in using it over Playground.

**Comment by penso** [comment]
Would that happen someday?
